### PR TITLE
Add Ransei

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

File: data/mods/ssb/statuses.ts [hunk 1]
```diff
@@ -367,6 +367,23 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
...
+		},
+		onSwitchOut() {
+			this.add(`c|${getName('Ransei')}|Switch Out Message`);
+		},
+		onFaint(pokemon) {
+			const target = pokemon.side.foe.active[0];
+			if (!target || target.fainted || target.hp <= 0) {
+				this.add(`c|${getName('Ransei')}|Ahah yes you got rekt! Welcome to Hackmons! gg m8!`);
+			} else {
+				this.add(`c|${getName('Ransei')}|ripsei… Ok look you might’ve won this time but I kid you not you’re losing next game!`);
```
Comment (KrisXV) [inline data/mods/ssb/statuses.ts]:
```suggestion
				this.add(`c|${getName('Ransei')}|ripsei... Ok look you might’ve won this time but I kid you not you’re losing next game!`);
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -862,6 +862,31 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
 
+	// Ransei
+	ripsei: {
+		num: 515,
+		accuracy: 100,
+		basePower: 0,
+		damageCallback(pokemon) {
+			const damage = pokemon.hp;
+			const target = pokemon.side.foe.active[0];
+			if (target && target.hp <= 0) target.faint();
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: this feels rather hacky, I would rather have the same effects from final gambit, and then have a check for fainting in an onAfterMove function (see: cant say's ability)

Comment (ghost) [reply data/mods/ssb/moves.ts]: also if there was a sturdy mon, they would be cheated out of their 1 hp.

Comment (Spandamn) [reply data/mods/ssb/moves.ts]: turns out this isnt gonna work, im unclaiming this one

**Comment by ghost** [comment]
Im gonna fix this

**Pull request merged**